simplemonitor/html/header.html

```diff
@@ -36,6 +40,33 @@
 			<div class="navbar-text" id="refresh_status"><span class="badge badge-warning d-none" id="refresh_badge">OUT OF DATE</span></div>
 			<div class="navbar-text">_COUNTS_</div>
 		</div>
+	<div id="mapid" style="fill: 1440px; height: 960px;"></div>
+	<script>
+
+		var mymap = L.map('mapid', {
+					center : [47.421583, -125.263214], 
+					zoom : 7.6
+		});
+	
+	
+		L.tileLayer('https://api.mapbox.com/styles/v1/{id}/tiles/{z}/{x}/{y}?access_token=pk.eyJ1IjoiZGFuaWVsZGgyMDYiLCJhIjoiY2szcmpnb2ExMDE2bDNkdDZyanYxcmF4bCJ9.qTBXFkdfzveo6kn3rsBmaQ', {

```

Comment:
⚠️ This token probably shouldn't be commited, you might want to generate yourself a new one.

Reply:
Good point about the token and I just created a new token. And it was a few months since I had done any updates with git so the pull request did get messy.